lib/internal/cluster/worker.js

```diff
@@ -3,8 +3,7 @@ const EventEmitter = require('events');
 const internalUtil = require('internal/util');
 const util = require('util');
 const defineProperty = Object.defineProperty;
-const suicideDeprecationMessage =
-    'worker.suicide is deprecated. Please use worker.exitedAfterDisconnect.';
+const deprecationMessage = 'Please use worker.exitedAfterDisconnect instead.';
```

Comment:
That's the paradox of this whole situation... see (content warning) https://github.com/Unitech/pm2/issues/2780